lib/api/v1/controllers/results.go

```diff
@@ -18,13 +18,10 @@ type ResultsController struct {
 }
 
 func (c *ResultsController) List(ctx *gin.Context) {
-	if c.EnsureAuthenticatedUser(ctx) != nil {
-		return
-	}
-
 	keyword := ctx.Query("keyword")
+	user := ctx.MustGet("CurrentUser").(*models.User)
```

Comment:
Should we create a function to get the current user from context? As we will use it a lot.

Reply:
Yes, added in [87bb8a0](https://github.com/carryall/go-google-scraper-challenge/pull/127/commits/87bb8a0e5fc2ed44534cee8403cd7d37ff081d09)